src/Acuminator/Acuminator.Tests/Tests/Utilities/AttributeInformation/AttributeInformationTests.cs

```diff
@@ -142,9 +190,9 @@ private void TestListOfParentsAggregate(string source)
 
 		[Theory]
 		[EmbeddedFileData(@"AggregateRecursiveAttributeInformation.cs")]
-		private void TestListOfParentsAggregateRecursive(string source)
+		private async void TestListOfParentsAggregateRecursiveAsync(string source)
```

Comment:
`Async void` is a bad practice. See [this article](http://andrey.moveax.ru/post/csharp-do-not-user-async-void) for more details.
Use `Task` instead - and you don't need `await` here, just `return`.